### PR TITLE
chore(examples) add nuxt example

### PR DESCRIPTION
**Pull request by lucasfernog**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] New Binding Issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes. Issue #___
- [x] No

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch and _not_ the `master` branch
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix: #xxx[,#xxx]`, where "xxx" is the issue number)

If adding a **new feature**, the PR's description includes:
- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a suggestion issue first and wait for approval before working on it)

**Other information:**

### CODE REVIEW

File: examples/vue/nuxt-app/nuxt.config.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
+
+export default {
+  mode: 'spa',
+  /*
```
Comment (thetre97) [inline examples/vue/nuxt-app/nuxt.config.js]: Could I suggest that you either add `server: { port: 4000 }` here to match the tauri conf, or edit the tauri conf to ``"devPath": "http://localhost:3000"` (the default Nuxt dev port)?

**Comment by jbolda** [comment]
@lucasfernog what is the status on this? Would be nice to merge for #353. Need someone to take it to the finish line?

**Comment by lucasfernog** [comment]
There's still an issue with nuxt + noserver mode :(

**Comment by lucasfernog** [comment]
@jbolda fixed it. gonna merge after actions checks.

**Pull request merged**